### PR TITLE
react, react-domをpeerDependenciesに変更

### PR DESCRIPTION
**Pull request by potaracom**
[Quick Start React](https://kintone.github.io/kintone-ui-component/latest/Getting-Started/QuickStart-React/)の手順に従いDropdownを表示すると以下のエラーが発生します。

Uncaught Invariant Violation: Invalid hook call. Hooks can only be called inside of the body of a function component. This could happen for one of the following reasons:
1. You might have mismatching versions of React and the renderer (such as React DOM)
2. You might be breaking the Rules of Hooks
3. You might have more than one copy of React in the same app
See https://fb.me/react-invalid-hook-call for tips about how to debug and fix this problem.

標題の対応でエラーが解消することを確認しました。

### CODE REVIEW

**Comment by phongnhat19** [comment]
@potaracom FYI, I've rebased this pull request into out developing branch and currently working on it.

File: package.json [hunk 1]
```diff
@@ -29,12 +29,14 @@
...
-		"react-dom": "16.8.6",
 		"@mdi/js": "3.6.95",
 		"@mdi/light-js": "0.2.63",
 		"@babel/polyfill": "7.2.5"
 	},
+	"peerDependencies": {
+		"react": "16.8.6",
+		"react-dom": "16.8.6"
+	},
 	"devDependencies": {
```
Comment (phongnhat19) [inline package.json]: Please add react and react-dom to devDependencies

**Comment by phongnhat19** [comment]
@potaracom Please have a look at my review. There's a minor change that need to be done.

**Comment by potaracom** [comment]
OK, add react and react-dom to devDependencies.

**Comment by phongnhat19** [comment]
@potaracom Thanks. I've merged your code into our code base and will consider releasing it in the next version

**Comment by phongnhat19** [comment]
@potaracom 
I think there's something confused here. We still need to keep `react` and `react-dom` in `peerDependencies`. We will update it in our dev branch, this is just to keep you up with our dev process.

**Pull request merged**